pkg/k8s/client.go

```diff
@@ -21,6 +22,9 @@ const (
 	getPodListSleepTimeMilliseconds = 300
 )
 
+// We only want to allow pod-identity with Pending or Running phase status
+var IGNORED_POD_PHASE_STATUSES = []string{"Succeeded", "Failed", "Unknown", "Completed", "CrashLoopBackOff"}
```

Comment:
Can you please move this to different casing ? 
May be: var ignoredPodPhases = []string {...}